script/integration-test

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+
+# subshell so that we can export PATH without breaking other things
+(
+	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start")
+	
+	# we need to wrap up everything in between integration-daemon-start and
```

Comment:
I **think** you could use `trap`: It calls some code before exiting the script (even if it was Ctrl-C'ed)